lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -42,8 +42,9 @@ __PACKAGE__->add_columns(
         data_type      => 'integer',
         is_foreign_key => 1,
     },
-    hidden => {
-        data_type     => 'boolean',
+    flags => {
```

Comment:
Indeed, I'll update commit message as soon as I'll get to some computer with actual keyboard.